src/components/Navbar.jsx

```diff
@@ -1,64 +1,38 @@
-    dispatch(reset());
-    navigate("/");
-  };
+import React, { useState } from 'react';
+import { withTranslation } from 'react-i18next';
+import logo from '../images/logo.png';
+import { MenuIcon, XIcon } from '@heroicons/react/outline';
+import SelectLanguage from './SelectLanguage';
+import { Link } from 'react-router-dom';
 
+function Navbar({t}) {
   const [dropDownMenu, setUpDropDown] = useState(false);
-  const [language, setLanguage] = useState("ENG");
 
   const handleClick = (e) => {
     e.preventDefault();
     setUpDropDown(!dropDownMenu);
   };
 
   return (
-    <div className="w-screen h-[90px] z-10 border-b-2 border-button-color fixed">
-      <div className="pl-[10%] flex justify-between items-center w-full h-full">
+    <div className="w-screen h-[90px] z-10  bg-[#000] bg-opacity-[12%] fixed drop-shadow-lg">
+      <div className="flex-none justify-between items-center w-full h-full">
         <div className="flex items-center w-full justify-between">
-          <img src={logo} alt="/" className="w-36" />
-          <div className="flex items-center">
-            <div className="md:pr-[90px]">
-              <ul className="hidden md:flex text-button-color font-semibold ">
-                <li>
-                  <Link to="/">
-                    <a>{t("navbar.home")}</a>
-                  </Link>
-                </li>
-                <li>{t("navbar.About")}</li>
-                <li>{t("navbar.routes")}</li>
-                <li>{t("navbar.contact")}</li>
+          <img src={logo} alt="/" className="w-36 pl-10" />
+          <div className="md:flex items-center">
+            <div className="md:pr-[50px]">
+              <ul className="hidden md:flex text-white font-semibold ">
+              <li>{t('navbar.home')}</li>
+                <li>{t('navbar.About')}</li>
+                <li>{t('navbar.routes')}</li>
+                <li className="whitespace-nowrap">{t('navbar.contact')}</li>
               </ul>
             </div>
             <div className="hidden md:flex h-[30px] pr-[70px]">
-              {user ? (
-                <button
-                  onClick={onLogout}
-                  className="w-[90px] border-white text-whte bg-red-700 hover:bg-button-color hover:text-white hover:border-none"
-                >
-                  {t("navbar.signOut")}
-                </button>
-              ) : (
-                <>
-                  <button className="w-[90px] border-white text-white bg-cyan-700 hover:bg-button-color hover:text-white hover:border-none">
-                    {t("navbar.signIn")}
-                  </button>
-                </>
-              )}
+              <button className="w-[90px] border-button-color border-fon text-white bg-transparent hover:bg-button-color hover:text-white hover:border-none">
+                <Link to='/login'>
+                <a>{t('navbar.signIn')}</a>
+                </Link>
+              </button>
               <div className="px-10">
```

Comment:
We still need the conditional rendering of the logout/signin button to remind the user if they try to access the site when they are still logged in. Please restore that

Reply:
logout added